.github/workflows/chart-manifest.yaml

```diff
@@ -0,0 +1,73 @@
+name: Chart manifest
+on:
+  pull_request:
+    branches:
+      - main
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  check_chart_manifest_in_sync:
+    if: github.event_name == 'pull_request'
```

Comment:
Thanks for your PR 👍 

Could you please separate these two tests in two files:
- `check-chart-manifest.yaml` with `check_chart_manifest_in_sync` job inside only
- `update-chart-manifest.yaml` with `sync_chart_manifest_on_comment` job inside only
It could avoid some `if` :)

Reply:
Seems to make sense. I think I just kept it in the same file so it was all related to manifest changes within the same spec but splitting indeed would avoid the if 👍